Sources/InstantSearchSwiftUI/View/HitsList.swift

```diff
@@ -68,42 +68,83 @@
     }
   }
 
-  #if os(iOS)
```

Comment:
By removing this conditional compiling clause, you make it fail while compiling for `macOS`. So, the `test` step [fails](https://github.com/algolia/instantsearch-ios/actions/runs/4015934909/jobs/6898306992).